public-images/public-images-how-to/index.rst

```diff
@@ -2,10 +2,11 @@
 
 How-to guides
 =============
-These guides provide instructions for performing basic operations related to Vagrant.
+These guides provide instructions for performing basic operations related to public Ubuntu cloud images.
 
 - :ref:`vagrant-bartender`
 - :ref:`run-a-vagrant-box`
+- :ref:`use-local-ds`
```

Comment:
At the end of this file (under the section toctree section) add: 
`Use a local cloud-init datasource <use-local-ds>`